java/connector-node/risingwave-connector-service/src/main/resources/mysql.properties

```diff
@@ -17,6 +17,9 @@ table.include.list=${database.name}.${table.name}
 include.schema.changes=${debezium.include.schema.changes:-false}
 database.server.id=${server.id}
 
+# default to UTC
+database.connectionTimeZone=UTC
```

Comment:
https://dev.mysql.com/doc/refman/8.0/en/time-zone-support.html#:~:text=Named%20time%20zones%20can%20be%20used%20only%20if
I am not sure if it is okay to assume the time zone info is always populated. Maybe '+00:00' is a safer option.

Reply:
Debezium only has `connectionTimeZone`, there is no way to set `forceConnectionTimeZoneToSession` and it seems unnecessary.
https://github.com/debezium/debezium/blob/8e269e358043cca28057091bfb0c341cd1e91aaa/debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnection.java#L486-L488